pkg/reconciler/revision/resources/deploy.go

```diff
@@ -170,6 +175,17 @@ func makePodSpec(rev *v1alpha1.Revision, loggingConfig *logging.Config, tracingC
 	return podSpec, nil
 }
 
+func verifySecrets(rev *v1alpha1.Revision, secretLister v1.SecretLister) error {
+	for _, volume := range rev.Spec.Volumes {
+		if secret := volume.VolumeSource.Secret; secret != nil {
+			if _, err := secretLister.Secrets(rev.Namespace).Get(secret.SecretName); err != nil {
+				return err
```

Comment:
This doesn't conform with @nimakaviani's earlier comment about collecting **all** missing secrets. Do we also only want to care about `NotFound` errors here? For others, we can't really tell what's going on, can we?

Reply:
That's true, but should we return the errors anyways?